CenterForOpenScience/ember-osf-web
# [ENG-4012] Un-silence deprecation manager-capabilities.modifiers-3-13

-   Ticket: [ENG-4012]
-   Feature flag: n/a

## Purpose

Un-silence the deprecation `manager-capabilities.modifiers-3-13` and fix the resulting problems

## Summary of Changes

1. Unsilence deprecation
2. Upgrade ember-auto-import to v2
3. Upgrade ember-modifiers
4. Upgrade ember-moment and imports
5. Spread moment around all the engines
6. Refactor modifiers to work with new api for ember-modifiers
7. Remove unused modifier
8. Fix usage of side-effect from ember computed to give ember power select a placeholder in institutional dashboard

## Side Effects

If I didn't fix the modifiers right, something may have broken there silently.

## QA Notes

1. Make sure that a modal still works (any modal is fine)
2. Make sure branding still has different colors/logos on things
3. Make sure dates/times display properly
4. Do google events still get sent for scrolling down the logged-out home page?
5. Does the Institutional Dashboard's department filter still work decently


[ENG-4012]: https://openscience.atlassian.net/browse/ENG-4012?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
